src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+import { themeGet, position } from 'styled-system';
+import { triangle } from 'polished';
+import { Box, NakedButton, Icon } from '../';
+
+const pointerWidth = '16px';
+const pointerHeight = '8px';
+
+const getPointerDimensions = (direction) => {
+  return direction === 'top' || direction === 'bottom'
+    ? {
+        width: pointerWidth,
+        height: pointerHeight,
+      }
+    : {
+        width: pointerHeight,
+        height: pointerWidth,
+      };
+};
+
+const getColor = (colorMode, theme) => {
+  if (theme) {
+    return colorMode === 'light'
+      ? theme.colors.mint
+      : theme.colors.greys.charcoal;
+  }
+  return themeGet(
+    colorMode === 'light' ? 'colors.mint' : 'colors.greys.charcoal',
+  );
+};
+
+const TooltipWrapper = styled(Box)`
+  ${position};
+  display: inline-block;
+  white-space: nowrap;
+  border-radius: ${themeGet('radii.default')};
+  background-color: ${({ colorMode }) => getColor(colorMode)};
+  padding: ${themeGet('space.2')} ${themeGet('space.4')};
+  &:after {
+    content: '';
+    position: absolute;
+    ${({ colorMode, pointingDirection, pointerPosition, theme }) => css`
+      ${triangle({
+        pointingDirection,
+        ...getPointerDimensions(pointingDirection),
+        foregroundColor: getColor(colorMode, theme),
+      })}
```

Comment:
Please add a semicolon here otherwise it might break formatting. 

```suggestion
      })};
```

In general, add a semicolon after every mixin block.

Reply:
👍